autoPyTorch/ensemble/ensemble_builder.py

```diff
@@ -1419,32 +1414,33 @@ def _add_ensemble_trajectory(self, train_pred: np.ndarray, test_pred: np.ndarray
     def _get_list_of_sorted_preds(self) -> List[Tuple[str, float, int]]:
         """
             Returns a list of sorted predictions in descending order
-            Scores are taken from self.read_scores.
+            Losses are taken from self.read_losses.
+
             Parameters
             ----------
             None
+
             Return
             ------
             sorted_keys: list
         """
-        # Sort by score - higher is better!
-        # First sort by num_run
-        sorted_keys = list(reversed(sorted(
+        # Sort by loss - smaller is better!
+        sorted_keys = list(sorted(
             [
-                (k, v["ens_score"], v["num_run"])
-                for k, v in self.read_scores.items()
+                (k, v["ens_loss"], v["num_run"])
+                for k, v in self.read_losses.items()
             ],
-            key=lambda x: x[2],
-        )))  # type: List[Tuple[str, float, int]]
-        # Then by score
-        sorted_keys = list(reversed(sorted(sorted_keys, key=lambda x: x[1])))
+            # Sort by loss as priority 1 and then by num_run on a ascending order
+            # We want small num_run first
+            key=lambda x: (x[1], x[2]),
+        ))
         return sorted_keys
 
     def _delete_excess_models(self, selected_keys: List[str]) -> None:
         """
             Deletes models excess models on disc. self.max_models_on_disc
             defines the upper limit on how many models to keep.
-            Any additional model with a worst score than the top
+            Any additional model with a worst loss than the top
```

Comment:
I think it should be a worse loss than the top

Reply:
Thanks,  I just pushed a new commit with this.